src/main/java/com/spaceuptech/space_api/utils/Service.java

```diff
@@ -0,0 +1,89 @@
+package com.spaceuptech.space_api.utils;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.spaceuptech.space_api.proto.FunctionsPayload;
+import io.grpc.stub.StreamObserver;
+
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.atomic.AtomicReference;
+
+public class Service {
+    private Config config;
+    private String serviceName;
+    private final String uuid;
+    private Map<String, ServiceFunction> METHODS;
```

Comment:
There is nothing special about methods. Normal camelCase convention should work

Reply:
Yes. Right